shared/schemas/projects/create-project.schema.ts

```diff
@@ -8,19 +8,25 @@ import { z } from "zod";
 
 export const CreateProjectSchema = z.object({
   projectName: z
-    .string()
-    .min(3, {
-      message: "Name must contain at least 3 characters.",
-    })
-    .max(255, { message: "Name must be less than 255 characters" }),
+      .string()
+      .min(3, { message: "Name must contain at least 3 characters." })
+      .max(255, { message: "Name must be less than 255 characters" }),
   countryCode: z.string().min(3).max(3),
   ecosystem: z.nativeEnum(ECOSYSTEM),
   activity: z.nativeEnum(ACTIVITY),
   projectSizeHa: z.number().nonnegative(),
-  priceType: z.nativeEnum(PROJECT_PRICE_TYPE),
+  priceType: z.nativeEnum(PROJECT_PRICE_TYPE).optional(),
```

Comment:
I made this optional to not break anything, but it should not be required, as the ideal output for each project would be the create 2 rows, each 1 with a single value for PROJECT_PRICE_TYPE enum